### PR TITLE
added confirmation before delete droplet

### LINKED ISSUE #109: Confirm before delete droplet
Feature request: We need a confirmation before delete (or any other critical action) a droplet. 😰

### PR DESCRIPTION
**Pull request by colkito**
just add a confirmation message before exec a delete action.

based in: https://gist.github.com/albrow/5882501

fixes #109

### CODE REVIEW

**Comment by bryanl** [comment]
Hello, 

If this functionality were to change, it would be introduced with a boolean flag, `--force`, or something similar.

**Comment by colkito** [comment]
yes! you're right @bryanl 
i'll work on that

File: commands/droplets.go [hunk 1]
```diff
@@ -430,20 +470,32 @@ func RunDropletDelete(c *CmdConfig) error {
 	} else if len(c.Args) > 0 && tagName != "" {
 		return fmt.Errorf("please specify droplets identifiers or a tag name")
 	} else if tagName != "" {
-		return ds.DeleteByTag(tagName)
+		fmt.Println("WARNING: Are you sure? (yes/no)")
```
Comment (bryanl) [inline commands/droplets.go]: There's a `warn` func in errors.go for this purpose.

**Comment by bryanl** [comment]
Hello,

There appears to be CI failures, could you please address these?

I also have some notes:

* You are going to want to ensure that you can test this. To do so, you might want to create a method to swap out the confirmation. There is a func, `retrieveUserTokenFunc` that is used to accomplish this, and you can build something similar. 

* `doctl` was build to be embedded in scripts. I didn't see a flag to force the deletion.

**Comment by bryanl** [comment]
This has been opened for a while with no activity, I'm going to close it.

**Pull request closed without merging**